core/trino-main/src/main/java/io/trino/server/remotetask/HttpRemoteTask.java

```diff
@@ -119,6 +123,10 @@
     private final ContinuousTaskStatusFetcher taskStatusFetcher;
     private final DynamicFiltersFetcher dynamicFiltersFetcher;
 
+    private final DynamicFiltersCollector taskDynamicFiltersCollector;
```

Comment:
`taskDynamicFiltersCollector` -> `outboundDynamicFiltersCollector`

Reply:
pushed this now